package/install_linux.sh

```diff
@@ -1,4 +0,0 @@
-#!/bin/bash
-set -ev
-# Install pyenv
-curl -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer | bash
```

Comment:
This was just not used anymore, or what? Can't find any references to this file, so I guess it's okay to remove it.

Reply:
Yeah, this is a leftover from older versions of CI steps, not used any more.